.vscode/extensions.json

```diff
@@ -0,0 +1,13 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
```

Comment:
Please remove comments. JSON does not support comments.

Reply:
this is supported by vscode default.